BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -8,6 +8,11 @@ import Foundation
 /// When adding new identifiers, please respect alphabetical order.
 /// Sing the song if you must.
 public struct StandardImageIdentifiers {
+    // Icon size 8x8
+    public struct Tiny {
+        public static let chevronDown = "chevronDownTiny"
+    }
+
```

Comment:
I'm still new to the codebase and had no idea this file existed! :) This is great.